src/components/shared/ContentItem/ListTemplate.js

```diff
@@ -73,10 +90,33 @@ const ListTemplate = ({
         </div>
       </div>
       <div className={clsx('cu_item_info', { [dir]: true, 'with_actions': !!children })}>
+        {
+          withCUInfo && (
+            <>
+              <Ref innerRef={cuInfoRef}>
+                <Header
+                  as={size === 'big' || isMobileDevice ? 'h5' : 'h3'}
+                  className={clsx('cu_item_name', { 'show_part': !cuInfoShowAll })}
+                >
+                  {(unit && unit.name) || (source && source.name) || (tag && tag.label)}
```

Comment:
this can be shortened like: unit?.name, source?.name, tag?.label

Reply:
replaced